tekton-catalog/kubectl-wrapper/deploy/kubectl-deploy.yaml

```diff
@@ -25,7 +25,7 @@ spec:
     default: "false"
     description: Enable set owner reference for created resource.
   - name: image
-    default: aipipeline/kubeclient:0.8.0 # it is huge
+    default: aipipeline/kubeclient:0.9.0 # it is huge
```

Comment:
Thanks @Tomcli -- should this PR wait until you push that image to DockerHub?

Reply:
I'm not publishing any 0.9.0 image yet, need to confirm can I merge #512 before cutting 0.9.0 release. Hopefully we can get an answer tomorrow.

I just want you to review everything is correct on the SDK side.